.github/workflows/run-tests.yml

```diff
@@ -600,11 +598,9 @@ jobs:
       - name: Get job ID from workflow run
         id: get_job_id
         run: |
-          set -euo pipefail
+          set -eu
```

Comment:
It's the other way around, you want `pipefail` to ensure errors aren't silently ignored.

Reply:
And I would add a `--fail` to the `curl` call unless you prove that these calls are expected to occasionally fail.

Reply:
Decided to not do this and instead explicitly call out that we are okay with this failing.

Also change to `JOB_ID=${job_id:-unknown}` so we at least have a sentinel value.

Reply:
done